### PR TITLE
Update UG with help function, ingredient listing and reset functions

### LINKED ISSUE #3: Update the User Guide

### PR DESCRIPTION
**Pull request by WM71811**
… levels functions

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -45,27 +45,45 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 **:information_source: Notes about the command format:**<br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+  e.g. in `ingredient NAME`, `NAME` is a parameter which can be used as `ingredient milk`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [t/TAG]` can be used as `n/John Doe t/friend` or as `n/John Doe`.
```
Comment (yanbingtao) [inline docs/UserGuide.md:51]: I think we should still use this prefix, to distinguish the input is either a name or a tag or something else.

Same applies to below, line 57

File: docs/UserGuide.md [hunk 2]
```diff
@@ -45,27 +45,45 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
+### Ingredients Tracking
+#### List Ingredient Levels : `ingredient`
+* Prints the ingredient levels for all ingredient types retrieved from the database.
+
+Format: `ingredient list`
+
+* Prints the ingredient level for a particular type of ingredient which is specified by the user’s command.
+
+Format: ` ingredient single INGREDIENT_NAME`
```
Comment (yanbingtao) [inline docs/UserGuide.md]: there is a white space before "ingredient", I think it should be a typo and need to be removed?

File: docs/UserGuide.md (hunk 1 above)
Comment (ureshiiYing) [reply docs/UserGuide.md:51]: I agree.

Comment (WM71811) [reply docs/UserGuide.md:51]: thanks, resolved

File: docs/UserGuide.md (hunk 2 above)
Comment (WM71811) [reply docs/UserGuide.md]: thanks, resolved

File: docs/UserGuide.md [hunk 3]
```diff
@@ -45,27 +45,45 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
 
 Format: `help`
 
+### Ingredients Tracking
+#### List Ingredient Levels : `ingredient`
+* Prints the ingredient levels for all ingredient types retrieved from the database.
+
+Format: `ingredient list`
+
+* Prints the ingredient level for a particular type of ingredient which is specified by the user’s command.
```
Comment (ureshiiYing) [inline docs/UserGuide.md]: I think that this should be specified as a sub feature under list, so as to distinguish between all ingredients and a single ingredient.

**Review by ureshiiYing — COMMENTED** [review]
Other than the parts commented, overall LGTM.

File: docs/UserGuide.md (hunk 3 above)
Comment (WM71811) [reply docs/UserGuide.md]: thanks

**Pull request merged**